code/datums/black_market_sellable.dm

```diff
@@ -69,78 +71,361 @@ var/list/black_market_sellables = list()
+	name = "Corpus the Snake"
+	item = /mob/living/simple_animal/cat/snek/corpus
+	no_children = 1
+	desc = "sssSSSSsss"
+	demand_min = 1
+	demand_max = 1	
 	price_min = 100
 	price_max = 200
-	display_chance = 100
+	sps_chance = 15
+	display_chance = 35
+	teleport_modifier = 2
+	
+/datum/black_market_sellable/animals/runtime
+	name = "Runtime the Cat"
+	item = /mob/living/simple_animal/cat/Runtime
+	no_children = 1
+	demand_min = 1
+	demand_max = 1
+	price_min = 150
+	price_max = 200
+	sps_chance = 15
+	display_chance = 35
+	teleport_modifier = 2	
 
-/datum/black_market_sellable/weapons/energy_gun
-	name = "Energy Gun"
-	item = /obj/item/weapon/gun/energy/
-	no_children = 0
-	desc = "Paying double for fully charged weapons, but guns with no charge will be denied."
+/datum/black_market_sellable/animals/salem
+	name = "Salem the Neglected Cat"
+	item = /mob/living/simple_animal/cat/salem
+	no_children = 1
 	demand_min = 1
-	demand_max = 3
+	demand_max = 1
 	price_min = 100
+	price_max = 150
+	sps_chance = 15
+	display_chance = 35
+	teleport_modifier = 2		
+
+	
+	
+	
+/datum/black_market_sellable/misc
+	category = "Miscellaneous Items"
+	
+/datum/black_market_sellable/misc/tome
+	name = "Occult Tome"
+	item = /obj/item/weapon/tome
+	no_children = 1
+	desc = "Looking for the one from the cult of Nar-Sie. Teleporting magical items will likely send an alert to Centcomm."
+	demand_min = 1
+	demand_max = 3
+	price_min = 350
+	price_max = 450
+	sps_chance = 75
+	display_chance = 60
+	
+/datum/black_market_sellable/misc/slimeheart
+	name = "Slime Heart"
+	item = /obj/item/slime_heart
+	no_children = 1
+	demand_min = 1
+	demand_max = 2
+	price_min = 150
 	price_max = 200
-	sps_chance = 100
-	display_chance = 100
+	sps_chance = 10
+	display_chance = 35
+
+/datum/black_market_sellable/misc/deathnettle
+	name = "Deathnettle"
+	item = /obj/item/weapon/grown/deathnettle
+	no_children = 1
+	desc = "Paying 1.5x if the potency is 80 or above."
+	demand_min = 3
+	demand_max = 6
+	price_min = 40
+	price_max = 60
+	sps_chance = 15
+	display_chance = 40
 	
-/datum/black_market_sellable/weapons/energy_gun/purchase_check(var/obj/input, var/mob/user)
-	if(istype(input,/obj/item/weapon/gun/energy/))
-		var/obj/item/weapon/gun/energy/gun = input
-		if(gun.power_supply.charge > 0)
-			return VALID
-	return "The energy gun does not have any charge."
+/datum/black_market_sellable/misc/deathnettle/determine_payout(var/obj/input, var/mob/user, var/payout)
+	if(istype(input,/obj/item/weapon/grown/deathnettle))
+		var/obj/item/weapon/grown/deathnettle/nettle = input
+		if(nettle.potency >= 80)
+			return payout*1.5
+	return payout	
 	
-/datum/black_market_sellable/weapons/energy_gun/determine_payout(var/obj/input, var/mob/user)
-	if(istype(input,/obj/item/weapon/gun/energy/))
-		var/obj/item/weapon/gun/energy/gun = input
-		if(gun.power_supply.charge >= gun.power_supply.maxcharge)
-			return get_price()*2
-	return get_price()
-*/
+/datum/black_market_sellable/misc/poutine
```

Comment:
I feel like this kind of takes away the specialness of cooking a megaproject if you can just order one.